apollo-router/src/uplink/license_enforcement.rs

```diff
@@ -214,18 +214,8 @@ impl LicenseEnforcementReport {
 
         let mut schema_violations: Vec<SchemaViolation> = Vec::new();
 
-        for subgraph_url in schema
-            .definitions
-            .iter()
-            .filter_map(|def| def.as_enum_type_definition())
-            .filter(|def| def.name == "join__Graph")
-            .flat_map(|def| def.values.iter())
-            .flat_map(|val| val.directives.iter())
-            .filter(|d| d.name == "join__graph")
-            .filter_map(|dir| (dir.arguments.iter().find(|arg| arg.name == "url")))
-            .filter_map(|arg| arg.value.as_str())
```

Comment:
Auto-merge happened before I saw this, sorry! Is connectors merging very soon? If this PR is an obstacle it could be reverted long enough to merge connectors and I’ll manage the conflict later.

Reply:
This PR was merged to dev but not forward ported to next, right?